sgx/Cargo.toml

```diff
@@ -18,8 +20,10 @@ enumerate  = { path = "../enumerate" }
 intel-types = { path = "../intel-types" }
 memory = { path = "../memory" }
 openssl = { version = "=0.10.29", features = ["vendored"], optional = true }
+iocuddle_lib = { path = "../iocuddle", optional = true, package = "iocuddle" }
```

Comment:
I suggested renaming the module instead below, which obviates the need for the awkward renaming here.

Reply:
Oh, sorry, I've already got rid of this. If you refresh the changes you'll see the rename is no longer needed.

Reply:
One style thing: we do merges of logical commits. We don't do stream-of-conciousness style PRs. So if you make this change here and then undo it in a later patch set, we're going to expect that these get squashed (as appropriate).

Reply:
Ah, ok. I wasn't sure cause every project is governed by its own policy. Would it be OK if I rebased selectively as well then? FWIW I'm personally a big fan of rebases and squashes but sometimes selective/retrospective rebases especially after the PR was submitted may lead to weirdness on Github. Let me know what you think @npmccallum!

Reply:
Yes, we are a fan of rebases and squashes. FYI, GitHub recently fixed the biggest issue I've seen with rebase and squash. https://github.blog/changelog/2020-07-14-pull-request-commits-now-ordered-chronologically/

Reply:
Rebased and squashed!